_posts/yiyangshi/2016-02-29-YiyangsPeerReview.md

```diff
@@ -0,0 +1,15 @@
+---
+author: yiyangshi
+layout: post:
```

Comment:
@yiyangshi this colon is preventing you YAML metadata from being read correctly.